pipenv/cli/command.py

```diff
@@ -760,24 +760,24 @@ def requirements(state, dev=False, dev_only=False, hash=False):
     for i, package_index in enumerate(lockfile["_meta"]["sources"]):
         prefix = "-i" if i == 0 else "--extra-index-url"
         echo(" ".join([prefix, package_index["url"]]))
+
+    from pipenv.utils.dependencies import convert_deps_to_pip
```

Comment:
Any particular reason we import this inline here and not at the top of the file?

Reply:
Because it's not used outside of this function.
This makes the import a bit more lazy, and pipenv startup a few milliseconds faster.

Reply:
This is also the case for all other subcommands which use local scope imports.